src/visualization/utilities.jl

```diff
@@ -481,6 +481,18 @@ function get_unstructured_data(u, solution_variables, mesh, equations, solver, c
     return unstructured_data
 end
 
+# This method is only for plotting 1D functions
+function get_unstructured_data(func::Function, solution_variables, mesh, equations,
```

Comment:
Does it make sense to ensure the 1D functionality by restricting e.g. the `mesh` to 1D meshes ?

Reply:
Makes sense. Did that in [c71a907](https://github.com/trixi-framework/Trixi.jl/pull/2250/commits/c71a907c4ee74ee07c84a208cc0b722c936e5e04).